desktop_version/src/Entity.cpp

```diff
@@ -1048,7 +1050,14 @@ void entityclass::createblock( int t, int xp, int yp, int w, int h, int trig /*=
             trig=0;
             break;
         case 35:
-            block.prompt = "Press ENTER to activate terminal";
+            if (custom)
+            {
+                block.prompt = "Press ENTER to interact";
+            }
+            else
+            {
+                block.prompt = "Press ENTER to activate terminal";
+            }
```

Comment:
These need to be changed to be `"Press %s to interact"` and `"Press %s to activate terminal"`, due to the new interact button option.